sys-apps/pcsc_lite/patches/pcsc_lite-1.8.25.patchset

```diff
@@ -0,0 +1,23 @@
+From 8e30b97b0d75b8ff067d2b94df7e064fef17c82c Mon Sep 17 00:00:00 2001
+From: Panagiotis Vasilopoulos <hello@alwayslivid.com>
+Date: Fri, 17 Jan 2020 14:44:43 +0000
+Subject: Imported network library
+
+
+diff --git a/configure.ac b/configure.ac
+index 0fa5168..a6f8660 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -147,6 +147,9 @@ PCSCLITE_FEATURES="${PCSCLITE_FEATURES} $PCSC_ARCH $host"
+ # See if socket() is found from libsocket
+ AC_CHECK_LIB(socket, socket, [LIBS="$LIBS -lsocket"])
+ 
++# Check for the network library.
++AC_CHECK_LIB(socket, socket, [LIBS="$LIBS -lnetwork"])
```

Comment:
Try with AC_SEARCH_LIBS instead

Reply:
Does that replace any functionality within the recipe?

Reply:
The idea is to pass the lib to configure with autoreconf

Reply:
Would this look alright?

![image](https://user-images.githubusercontent.com/30193966/72659596-e8322980-39ca-11ea-9b3a-af6c8c8f5e1c.png)

Reply:
Look at other recipes how they use it, it bit overkill here :)

Reply:
Actually this works with a small change:
```diff
-+AC_CHECK_LIB(socket, socket, [LIBS="$LIBS -lnetwork"])
++AC_CHECK_LIB(network, socket, [LIBS="$LIBS -lnetwork"])
```

Reply:
Seems surprising enough to me. I pushed the changes, but I'll test it out in an hour.